splunk_enterprise_security_tag_assets_and_identities.py

```diff
@@ -0,0 +1,469 @@
+"""
+Collects possible user and system data types and checks Splunk Enterprise Security for asset and identity data. If there is a match, it will tag the indicator record with &quot;asset&quot; or &quot;identity.&quot;
```

Comment:
I think we either need to change the tags referenced here or change the tag blocks, which use the tags "known_identity" and "known_asset". Up to you which tags to use.